api-specificatie/openapi.yaml

```diff
@@ -320,13 +303,7 @@ paths:
       operationId: raadpleegAdresseerbaarobject
       tags:
         - Adresseerbaar object
-      summary: Raadpleeg adresseerbaar object met een identificatie van een nummeraanduiding of pand object
-      description: "Raadpleeg het adresseerbaar object met de identificatie van een nummeraanduiding of pand via de zoek-operatie.
-        Geleverd worden de huidige gegevens van een adresseerbaarobject evt. met adres en pand.
-        Huidige gegevens zijn actuele gegevens die geen eind status hebben, zoals 'Naamgeving ingetrokken'.
-        Als de nummeraanduiding is ingetrokken, dan levert een bevraging geen adres op.
-        Parameter nummeraanduidingidentificatie-query bevat de 16 cijferige identificatie van een nummeraanduiding.
-        Parameter pandidentificatie-query bevat de 16 cijferige identificatie van een pand"
+      description: "Zoek een adresseerbaar object met een adresidentificatie, nummeraanduidingidentificatie of pandidentificatie."
```

Comment:
Zoek een adresseerbaar object. Een adresseerbaar object is een verblijfsobject, ligplaats of standplaats waaraan adressen kunnen worden toegekend.